src/Components/RightClick/Base/RightClick.scss

```diff
@@ -0,0 +1,98 @@
+@import '../../../Styles/modules';
```

Comment:
nit: new to clockface, but why aren't we using absolute positioning to make migrations and restructuring of folders easier during maintenance mode on the code base?

Reply:
We tried to set it up and failed, so it's on the back burner. If y'all can make it work we are very down